members/_posts/2016-10-04-lewic.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: memberpage
+category: cmember
+logo : lewic.png

```

Comment:
This image is fine for the member page, but by far too wide for the member list (which you created, so don't blame me that I complain here 😛).
I wonder if we should ask members to provide two images where we also define the constraints (max width, height, size in KB). WDYT?